src/main.ts

```diff
@@ -13,6 +13,44 @@ import { AxiosResponse } from 'axios';
 
 export { LunchMoneyCryptoConnection } from './types/lunchMoney';
 
+// According to https://support.kraken.com/hc/en-us/articles/360001206766-Bitcoin-currency-code-XBT-vs-BTC
```

Comment:
@lunchbag here is the updated logic that no longer uses the catchall, that you commented about.